
# obstacles parameters
max_frames: 30

#posibility threshold to accept an object
posibility_threshold: 0.7

msg_object_detection: /object_analytics/detection

msg_object_tracking: /object_analytics/tracking

msg_object_localization: /object_analytics/localization

